### PR TITLE
Script - SalesforceFindObject

### PR DESCRIPTION
**Pull request by yuvalbenshalom**
salesforce find object, replaces "Select *"

### CODE REVIEW

File: Scripts/script-SalesforceFindObjects.yml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+commonfields:
+  id: 22c04dd1-0334-4aaa-897a-851d85479533
+  version: 65
+name: SalesforceFindObjects
+script: |2
+
+
+
+  var res = executeCommand('salesforce-get-object', {path: args.object + "/describe"})[0];
+  var fields = res.Contents.fields;
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: What happens if res is an error? Or if results are empty?

File: Scripts/script-SalesforceFindObjects.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+commonfields:
+  id: 22c04dd1-0334-4aaa-897a-851d85479533
+  version: 65
+name: SalesforceFindObjects
+script: |2
+
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Why so many blank lines? :)

File: Scripts/script-SalesforceFindObjects.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+  var res = executeCommand('salesforce-get-object', {path: args.object + "/describe"})[0];
+  var fields = res.Contents.fields;
+  var fieldNames = [];
+
+  for (var key in fields) {
+    fieldNames.push(fields[key].name)
+  }
+
+  if (args.field && args.fieldValue){
+      return executeCommand('salesforce-query', {query: "SELECT+"+ fieldNames.join(',')+"+from+"+args.object+"+where+"+args.field+"='"+args.fieldValue+"'" });
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Scripts should return human readable tables/markdown not jsons

File: Scripts/script-SalesforceFindObjects.yml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  }
+  return executeCommand('salesforce-query', {query: "SELECT+"+ fieldNames.join(',')+"+from+"+args.object});
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Same regarding table/markdown

File: Scripts/script-SalesforceFindObjects.yml (hunk 3 above)
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Should also add results to context

File: Scripts/script-SalesforceFindObjects.yml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+type: javascript
+tags: []
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Salesforce tag?

File: Scripts/script-SalesforceFindObjects.yml [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+enabled: true
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Not needed. Add "system: true" instead

File: Scripts/script-SalesforceFindObjects.yml [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- name: object
+  required: true
+  default: true
+  description: Object Name
+- name: field
+  description: Field Name
+- name: fieldValue
+  description: Field Value
+scripttarget: 0
+dependson: {}
```
Comment (liorkol) [inline Scripts/script-SalesforceFindObjects.yml]: Should add commands that this depends on - see other scripts for exact syntax

**Comment by meirwah** [comment]
@yuvalbenshalom what is the status of this?

**Comment by liorkol** [comment]
@meirwah this is not ready. Leave it for now.

**Comment by meirwah** [comment]
@yuvalbenshalom @liorkol any update on this?

**Pull request closed without merging**